interface/forms/vitals/C_FormVitals.class.php

```diff
@@ -83,40 +99,54 @@ public function default_action()
         $this->assign("patient_dob", $patient_dob);
 
         $i = 1;
-        while ($result = sqlFetchArray($res)) {
-            $results[$i]['id'] = $result['id'];
-            $results[$i]['uuid'] = $result['uuid'];
-            $results[$i]['encdate'] = substr($result['encdate'], 0, 10);
-            $results[$i]['date'] = $result['date'];
-            $results[$i]['activity'] = $result['activity'];
-            $results[$i]['bps'] = $result['bps'];
-            $results[$i]['bpd'] = $result['bpd'];
-            $results[$i]['weight'] = $result['weight'];
-            $results[$i]['height'] = $result['height'];
-            $results[$i]['temperature'] = $result['temperature'];
-            $results[$i]['temp_method'] = $result['temp_method'];
-            $results[$i]['pulse'] = $result['pulse'];
-            $results[$i]['respiration'] = $result['respiration'];
-            $results[$i]['BMI'] = $result['BMI'];
-            $results[$i]['BMI_status'] = $result['BMI_status'];
-            $results[$i]['note'] = $result['note'];
-            $results[$i]['waist_circ'] = $result['waist_circ'];
-            $results[$i]['head_circ'] = $result['head_circ'];
-            $results[$i]['oxygen_saturation'] = $result['oxygen_saturation'];
-            $results[$i]['oxygen_flow_rate'] = $result['oxygen_flow_rate'];
-            $results[$i]['ped_weight_height'] = $result['ped_weight_height'];
-            $results[$i]['ped_bmi'] = $result['ped_bmi'];
-            $results[$i]['ped_head_circ'] = $result['ped_head_circ'];
+        // eventually we want this just to use the service search date but we will move this here.
+        $records = $vitalsService->getVitalsHistoryForPatient($GLOBALS['pid'], $form_id);
+
+        foreach ($records as $result) {
+            $historicalVitals = new FormVitals();
+            // TODO: @brady.miller note we dropped the inconsistency of displaying historical usa weights in lbs&oz and entering them in via decimal format.
+            // Everything now is consistent decimal format, I can put it back but its a bit of work to treat weight special over everything else and seems
+            // odd the inconsistency
+            $historicalVitals->populate_array($result);
+            $results[$i] = $historicalVitals;
             $i++;
         }
 
         $this->assign("vitals", $vitals);
         $this->assign("results", ($results ?? null));
 
+        $this->assign('interpretation_options', $this->interpretationsList);
+
         $this->assign("VIEW", true);
         return $this->fetch($this->template_dir . $this->template_mod . "_new.html");
     }
 
+    private function get_interpretation_list_options()
+    {
+        $listService = new \OpenEMR\Services\ListService();
```

Comment:
guessing this change not yet pushed to the PR

Reply:
Should be pushed now!